docusaurus.config.js

```diff
@@ -19,8 +19,8 @@ const config = {
     'Fast, easy and reliable testing for anything that runs in a browser.',
   url: 'https://docs.cypress.io',
   baseUrl: '/',
-  onBrokenLinks: 'throw', // TODO: update this to throw when we go live to production
-  onBrokenMarkdownLinks: 'throw',
+  onBrokenLinks: 'warn',
+  onBrokenMarkdownLinks: 'warn',
```

Comment:
Why were these changes to warn?

Reply:
Just so this could build without breaking for internal review purposes while the relative content URLs are in flux. 

This is a draft PR so most of the nuance isn't worth combing through until all feedback is incorporated and ready for 👀